packs/data/campaign-effects.db/effect-burned-tongue-linguistic.json

```diff
@@ -28,7 +28,7 @@
             }
         ],
         "source": {
-            "value": "Pathfinder #178: Punks in a Powderkeg"
+            "value": "Pathfinder #178: Punks in a Powder Keg"
```

Comment:
This one is awkward because the cover page of the PDF is 2 words but the name on Paizo.com is one.

https://paizo.com/products/btq02d61?Pathfinder-Adventure-Path-178-Punks-in-a-Powderkeg

In English it is properly two words though.

Reply:
Yeah, I waffled a bit on this too. In the end I treated the PDF as canonical.

Reply:
@TMunsie thoughts?